src/_includes/database_backups.md

```diff
@@ -1,15 +1,38 @@
-If your database is in a paid plan (i.e. it's not "free plan"), we'll
-automatically make periodic backups of your database on a daily basis, at around
-1:00 AM Central European Time (CET or UTC+0100). The time of your daily backup
-is configurable via the web dashboard of your database or using the CLI. The scheduled date is not strongly enforce: it might get delayed depending on the load on our infrastructure.
+If your database is in a paid plan (i.e. it's not "free plan"), we'll automatically make periodic backups of your database on a daily basis, at around 1:00 AM Central European Time (CET or UTC+0100). The time of your daily backup is configurable via the web dashboard of your database or using the CLI. The scheduled date is not strongly enforce: it might get delayed depending on the load on our infrastructure.
+
+#### Using the Database Dashboard
+
+1. From your web browser, open your database dashboard
+2. Click the **Backups** tab
+3. Locate the **Backup schedule** block
+4. Click the **Schedule** button
+5. Make sure to check the **I want to enable scheduled backups** checkbox
+6. Pick an hour (**timezone is UTC**)
+7. Validate by clicking the **Update** button
+
+#### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. Configure the time of backup:
+   - By setting an hour:
+     ```bash
+     scalingo --app my-app --addon [YOUR ADDON NAME] backups-config --schedule-at 3
+     ```
+     In this example, we ask the platform to create the backups at ~03:00.
+     _Note: The timezone used is the local timezone of the machine running the command._
+   - By setting an hour and a timezone:
+     ```bash
+     scalingo --app my-app --addon [YOUR ADDON NAME] backups-config --schedule-at "4:00 UTC"
```

Comment:
question: I'm wondering about `[YOUR ADDON NAME]`, I would have probably chosen another wording such as `[YOUR ADDON KIND]` or `[YOUR ADDON TYPE]`. I'll let you decide :)

Reply:
No strong feelings on this one, I didn't really know what to display as placeholder here so wil go with one of your first suggestion!